pandas/tests/series/test_dtypes.py

```diff
@@ -1,123 +1,21 @@
-from datetime import datetime, timedelta
-from importlib import reload
```

Comment:
what exactly is left over here? i am not sure i get why you are moving to a new file than just renaming and moving out anything that isn't astype (which is a very small number)

Reply:
9 tests left here for which i havent found a home, but it definitely isnt test_astype.  one probably belongs in the DataFrame tests, one for reindex, one for constructors...

i do expect this file to be killed off before too long

Reply:
ok its fine, just though would be easier to rename
but ok